crates/astria-conductor/src/block_verifier.rs

```diff
@@ -47,21 +50,21 @@ use tracing::{
 /// availability layer. `validate_sequencer_block` is used to validate the blocks received from
 /// either the data availability layer or the gossip network.
 pub struct BlockVerifier {
-    sequencer_client: Client,
+    sequencer_client: HttpClient,
 }
 
 impl BlockVerifier {
     pub fn new(sequencer_url: &str) -> eyre::Result<Self> {
         Ok(Self {
-            sequencer_client: Client::new(sequencer_url)
+            sequencer_client: HttpClient::new(sequencer_url)
                 .wrap_err("failed to construct sequencer client")?,
         })
     }
 
     /// validates `SignedNamespaceData` received from Celestia.
     /// This function verifies the block signature and checks that the data
     /// was signed by the expected proposer for this block height.
-    pub(crate) async fn validate_signed_namespace_data(
+    pub async fn validate_signed_namespace_data(
```

Comment:
why pub?

Reply:
This was probably an oversight undoing the `BlockVerifier as extension trait` change.